### PR TITLE
Book

### PR DESCRIPTION
**Pull request by AbdiazizH**
created templates for booking forms

### CODE REVIEW

**Review by Tajeu2001 — APPROVED** [review]
Remove the index view function and url if you're no longer using it.

File: customer/urls.py [hunk 1]
```diff
@@ -2,5 +2,8 @@
 from . import views
 
 urlpatterns = [
-  path('', views.index, name='custhome'),
-]
\ No newline at end of file
+  path('available', views.available, name='available'),
```
Comment (peterken674) [inline customer/urls.py:5]: You must add a slash at the end of the url otherwise it won't work.

**Review by peterken674 — CHANGES_REQUESTED** [review]
@AbdiazizH did you agree on a different design with @layersony? The forms don't look anything like the designs we did. Also we cannot merge with `virtual/`, you need to remove it.

File: customer/urls.py [hunk 2]
```diff
@@ -2,5 +2,8 @@
 from . import views
 
 urlpatterns = [
-  path('', views.index, name='custhome'),
```
Comment (layersony) [inline customer/urls.py:5]: this path is required for customer login

**Review by layersony — COMMENTED** [review]
make the changes

**Pull request merged**